src/types/request.rs

```diff
@@ -256,25 +256,55 @@ impl PyRequest {
 
     pub fn json(&self, py: Python) -> PyResult<PyObject> {
         match self.body.as_ref(py).downcast::<PyString>() {
-            Ok(python_string) => match serde_json::from_str(python_string.extract()?) {
-                Ok(Value::Object(map)) => {
-                    let dict = PyDict::new(py);
-
-                    for (key, value) in map.iter() {
-                        let py_key = key.to_string().into_py(py);
-                        let py_value = match value {
-                            Value::String(s) => s.as_str().into_py(py),
-                            _ => value.to_string().into_py(py),
-                        };
+            Ok(python_string) => json_string_to_pyobject(py, python_string.extract()?),
+            Err(e) => Err(e.into()),
+        }
+    }
+}
 
-                        dict.set_item(py_key, py_value)?;
-                    }
+///serde_json not impl IntoPy<PyObject>  but orphan principle. so use this function
```

Comment:
@asamaayako , what is the `orphan principle` ? Could you please explain?

Reply:
sorry, I means  `Orphan Rule`.  I can't  impl IntoPy<PyObject>  for serde_json  in this create.  because  either the trait IntoPy<PyObject> or the type serde_json must be defined in the current crate.